src/LaboratoryResults-3-0/Test/Receive/2-1b-receive-panelLOINC.xml

```diff
@@ -0,0 +1,186 @@
+            text = "Elise Erelies" 
+            given = "Elise" 
+            family = "Erelies"
+            urlfamily = "http://hl7.org/fhir/StructureDefinition/humanname-own-name"
+            urliso = "http://hl7.org/fhir/StructureDefinition/iso21090-EN-qualifier"
+            iso = "BR"
+          />
+        <nts:include value="assert.patient.gender" direction="response" resource="Patient"
+            url = "http://nictiz.nl/fhir/StructureDefinition/ext-CodeSpecification"
+            system = "http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender"
+            code = "F"
+            display = "Vrouw"
+        />
+        <nts:include value="assert.patient.birthdate" direction="response" resource="Patient" date="1950-11-01"
+        />
+
+<!-- Asserts for checking the panel Observation to be compliant with the wiki page on
+            https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten-->       
+        <nts:include value="assert.identifier" direction="response" resource="Observation" system="urn:oid:2.16.840.1.113883.2.4.3.11.999.25.4296.2" identifier="LAR_NierfunctiePanelLOINC"/>
+
+<!-- Asserts for checking the first Observation to be compliant with the wiki page on
+            https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten--> 
+        <nts:include value="assert.identifier" direction="response" resource="Observation" system="urn:oid:2.16.840.1.113883.2.4.3.11.999.25.4298.2" identifier="11111"/>
+        <nts:include value="assert.observation.valueq" direction="response" resource="Observation"
+            quantity = "97"
+            unit = "mL/min/{1.73_m2}"
+            system = "http://unitsofmeasure.org"
+            code="mL/min/{1.73_m2}"
+        />
+        <nts:include value="assert.observation.status" direction="response" resource="Observation"
+            status="final"
+        />
+        <nts:include value="assert.observation.referencerange" direction="response" resource="Observation" 
+            range="low"
+            rangevalue = "90"
+            unit= "mL/min/{1.73_m2}"
+            system="http://unitsofmeasure.org"
+            code="mL/min/{1.73_m2}"
+        />
+        <nts:include value="assert.observation.interpretation" direction="response" resource="Observation"
+            interpretation="Waarde is goed."
+        />
+ 
+<!-- Asserts for checking the second Observation to be compliant with the wiki page on
+            https://informatiestandaarden.nictiz.nl/wiki/Lab:1.0.0_Testgegevens_SturenResultaten-->     
+        <nts:include value="assert.identifier" direction="response" resource="Observation" system="urn:oid:2.16.840.1.113883.2.4.3.11.999.25.4298.2" identifier="22222"/>  
+        <nts:include value="assert.observation.valueq" direction="response" resource="Observation"
+            quantity = "135"
+            unit = "mmol/l"
+            system = "http://unitsofmeasure.org"
+            code="mmol/l"
```

Comment:
In UCUM a liter is represented by both _l_ and _L_. The unit millimole per liter however, is mentioned as _mmol/L_ on https://ucum.org/ucum. Might be worth a discussion with IA?